browser/resources/settings/brave_overrides/add_site_dialog.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at https://mozilla.org/MPL/2.0/.
+
+import {RegisterPolymerComponentBehaviors} from 'chrome://resources/polymer_overriding.js'
+import {BraveAddSiteDialogBehavior} from '../brave_add_site_dialog/brave_add_site_dialog_behavior.js'
+
+RegisterPolymerComponentBehaviors({
```

Comment:
I think it's going to be more future-proof to instead call `RegisterPolymerComponentReplacement`, since behaviors and polymer v1 / v2 are deprecated now. You would provide a new polymer component which uses the existing chromium component as a mixin. See here as an example https://github.com/brave/brave-core/blob/2de497cae9adc3cd12c5cf97000ef0f940c11acf/browser/resources/settings/brave_clear_browsing_data_dialog/brave_clear_browsing_data_dialog_behavior.ts#L16

Reply:
done